tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -932,6 +932,10 @@ tf_module {
     name: "clip_by_value"
     argspec: "args=[\'t\', \'clip_value_min\', \'clip_value_max\', \'name\'], varargs=None, keywords=None, defaults=[\'None\'], "
   }
+  member_method {
```

Comment:
move it before complex. The list needs to be sorted.

Reply:
Done. 

Do you know why the compatibility test doesn't work for V1?